opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -535,6 +536,12 @@ class SpanLimits:
       environment variable.
     - If the environment variable is not set, the default value for the limit is used.
 
+    Limit precedence:
+    
+    - If a model specific limit is set, it will be used.
+    - Else if the model specific limit has a default value, the default value will be used.
+    - Else if model specific limit has a corresponding global limit, the global limit will be used. 
```

Comment:
Shouldn't 543 and 542 be switched?

1. Model specific if set
2. Global specific if set
3. Model default
4. Global default

Reply:
They way I understood the spec, I think model default takes precedence over global user provided value and it sure is counter-intuitive. Spec says the following:

>  An SDK MAY implement model-specific limits, for example SpanAttributeCountLimit. If both a general and a model-specific limit are implemented, then the SDK MUST first attempt to use the model-specific limit, if it isn't set and doesn't have a default, then the SDK MUST attempt to use the general limit.

Am I reading it wrong? 

Also created a spec issue here to get clarification: https://github.com/open-telemetry/opentelemetry-specification/issues/1878